src/app/Job-Service/job.service.spec.ts

```diff
@@ -1,12 +1,60 @@
 import { TestBed } from '@angular/core/testing';
-
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
 import { JobService } from './job.service';
+import { of } from 'rxjs/internal/observable/of';
 
 describe('JobService', () => {
-  beforeEach(() => TestBed.configureTestingModule({}));
+  let jobService: JobService; // added for testing
+
+  beforeEach(() =>{
+    TestBed.configureTestingModule({
+    imports: [HttpClientTestingModule],
+    providers: [JobService] 
+  });
+  jobService = TestBed.get(JobService); //added for testing
+});
 
   it('should be created', () => {
-    const service: JobService = TestBed.get(JobService);
-    expect(service).toBeTruthy();
+    // const service: JobService = TestBed.get(JobService);
+    expect(jobService).toBeTruthy();
+  });
+  describe('getAllJobs', () => {
+    it('it should return all jobs', () => {
+      const jobResponse =[
+        {id: 1,
+        title: "Developer",
+        description: "test",
+        location: "Chicago",
+        isFilled: true,
+        skills: []
+        }];
+      let response;
+      spyOn(jobService, 'getAllJobs').and.returnValue(of(jobResponse));
+
+      jobService.getAllJobs().subscribe(res => {
+        response = res;
+      });
+      expect(response).toEqual(jobResponse);
+    });
+
   });
 });
+
```

Comment:
This test looks like it would also not actually be testing anything, if it weren't commented out. As is, why is this left in the code base as a comment? It should be either fixed or removed.